fri/src/prover/mod.rs

```diff
@@ -209,14 +223,20 @@ where
         // projection to reduce the degree of evaluations by N
         let alpha = channel.draw_fri_alpha();
         *evaluations = apply_drp(&transposed_evaluations, self.domain_offset(), alpha);
-
         self.layers.push(FriLayer {
             tree: evaluation_tree,
             evaluations: flatten_vector_elements(transposed_evaluations),
             _base_field: PhantomData,
         });
     }
 
+    /// Builds a last FRI layer by hashing sequentially `evaluations`.
+    fn build_remainder(&mut self, channel: &mut C, evaluations: &mut [E]) {
```

Comment:
nit: since we are not really building the remainder here - would a more appropriate name for this be `set_remainder()`?

Reply:
We can use your suggestion as I couldn't come up with a better name than `set_remainder`.

Reply:
Seems like this didn't get changed in the last commit?

Reply:
Yes, changed now. Thanks!